core/unix/signal_macos.c

```diff
@@ -158,11 +158,11 @@ sigcontext_to_mcontext_simd(priv_mcontext_t *mc, sig_full_cxt_t *sc_full)
      */
     ASSERT(MCXT_NUM_SIMD_SLOTS == proc_num_simd_registers());
     for (i = 0; i < proc_num_simd_registers(); i++) {
-        memcpy(&mc->ymm[i], &sc->__fs.__fpu_xmm0 + i, XMM_REG_SIZE);
+        memcpy(&mc->simd[i], &sc->__fs.__fpu_xmm0 + i, XMM_REG_SIZE);
```

Comment:
I assume you've done a search of the sources to ensure there are no further instances?

Reply:
Indeed, but w/o being able to build MacOS, I rely on grep -r "\->ymm" * and grep -r "\.ymm" *.